sample-apps/delayed_job-sample/README.md

```diff
@@ -26,11 +26,11 @@ Run `./bin/dev` to run the app in development mode. This will...
 
 ## How to use this sample app
 
-Open https://requestinspector.com/p/judoscale-ruby in a browser. The sample app is configured to use this endpoint as a mock for the Rails Autoscale Adapter API. This page will monitor all API requests sent from the adapter.
+Open https://judoscale-adapter-mock.requestcatcher.com in a browser. The sample app is configured to use this endpoint as a mock for the Judoscale Adapter API. This page will monitor all API requests sent from the adapter.
```

Comment:
request catcher was working out of the box for me, seemed easier to just swap it back.